ci/common-setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+function init_test {
+  cd
+  rm -rf /tmp/mypaper
+  mkdir /tmp/mypaper
+  cd /tmp/mypaper
+  git init
+  popper init
+}
+
+if USE_VIRTUALENV
+then
+  source /tmp/popper-env/bin/activate
```

Comment:
On line 8 of `.travis.yml`, we are adding the `cli/bin` folder to `PATH` which makes `cli/bin/popper` the binary being executed when `popper` is used subsequently. After the virtualenv gets activated, do you know which of the two available `popper` commands gets invoked? My guess is that the one from virtualenv has higher precedence but it might be better to double check so that we are certain

Reply:
I did test this out by running `which popper` in the script after sourcing the virtualenv, which indeed outputs `/tmp/popper-env/bin/popper`. I'll put the `which` in `setup.sh`, just for the sake of debugging.